DigitalLearningSolutions.Web/Controllers/NotificationPreferencesController.cs

```diff
@@ -15,6 +15,7 @@ namespace DigitalLearningSolutions.Web.Controllers
     [TypeFilter(typeof(ValidateAllowedDlsSubApplication))]
     [SetDlsSubApplication]
     [SetSelectedTab(nameof(NavMenuTab.MyAccount))]
+    [Authorize(Policy = CustomPolicies.CentreUser)]
     public class NotificationPreferencesController : Controller

```

Comment:
Should we be hiding the button on the My Account page for this?

Reply:
Content changes are coming in the later tickets that I left TODOs for.